source/renderer/app/themes/daedalus/cardano.js

```diff
@@ -700,8 +700,10 @@ export const CARDANO_THEME_OUTPUT = {
     '--rp-select-arrow-bg-color': 'rgba(94, 96, 102, 0.3)',
     '--rp-select-arrow-bg-color-open': 'rgba(94, 96, 102, 0.7)',
     '--rp-select-input-bg-color': 'transparent',
+    '--rp-select-input-bg-color-disabled': 'rgba(255, 255, 255, 0.05)',
```

Comment:
Same styling as above was implemented for dropdowns too (they have nearly similar design).